utils/utils.go

```diff
@@ -731,6 +740,19 @@ func CustomizeServiceMonitor(sm *prometheusv1.ServiceMonitor, ba common.BaseComp
 		sm.Spec.Endpoints[0].MetricRelabelConfigs = endpoints[0].MetricRelabelConfigs
 		sm.Spec.Endpoints[0].HonorTimestamps = endpoints[0].HonorTimestamps
 		sm.Spec.Endpoints[0].HonorLabels = endpoints[0].HonorLabels
+	}
+	if ba.GetManageTLS() == nil || *ba.GetManageTLS() {
```

Comment:
ServiceMonitor is always created from scratch, so no clean-up is needed. Just wanted to confirm if that's the case?